content/blog/2025-11-05-valkey-helm-chart/index.md

```diff
@@ -0,0 +1,206 @@
+
+If your pipelines pull Bitnami charts or images during deploys, you may experience significant operational issues: rollouts can fail with `ImagePullBackOff` or auth/404 errors, clusters can drift when staging keeps old cached images while production can't pull or resolves a different tag, and "invisible" upgrades can occur when a moved tag points to a new digest. During incidents, rollbacks may slow down or fail entirely because the old image isn't fetchable.
+
+Beyond deployment issues, compliance can break, security patches can stall behind limits or paywalls, and you may face surprise licensing or mirroring costs. The net effect: slower releases, harder debugging, inconsistent environments, and higher operational and business risk.
+
+To reduce the impact on Valkey deployments, the community created an official, project-maintained Helm chart (request: [issue #2371](https://github.com/valkey-io/valkey/issues/2371), chart: [valkey-helm](https://github.com/valkey-io/valkey-helm)). With the official chart, you can pin chart and image versions, keep `values.yaml` in code, and upgrade on your schedule without depending on vendor policy changes.
+
+## Why a Valkey maintained chart helps
+
+With the official chart, you control exactly which versions you deploy, without third-party vendor policies forcing unexpected changes. Pin a chart release from the Valkey repo (for example `--version 0.9.0` from [https://github.com/valkey-io/valkey-helm](https://github.com/valkey-io/valkey-helm)) and lock the Valkey image tag in your `values.yaml`. Because the chart follows Valkey releases and docs, you can bump versions in a pull request, test in staging, then promote the same versions to production.
+
+## Essential capabilities in the Valkey Helm Chart
+
+The official Valkey Helm chart supports the following:
+
+* **Standalone cache** - Deploy a single Valkey instance with or without data persistence, perfect for simple caching layers and development environments.
+
+* **Replicated read-heavy workloads** - Use a primary-replica topology with separate read and read-write endpoints, distributing read traffic across all replica instances while routing writes to the primary node.
+
+* **ACL-based authentication** - Enable authentication using Access Control Lists for fine-grained user permissions and password-based authentication.
+
+* **TLS encryption** - Enable TLS for encrypted client-server and replica-primary communication, protecting data in transit.
+
+* **Metrics** - Collect Valkey metrics by enabling the Prometheus exporter.
```

Comment:
Include a link to the exporter, otherwise this isn't very clear. Also, how exactly does the helm chart support this?